### DISCUSSION TITLE
Students cannot download repositories.

### DISCUSSION THREAD

**Discussion by mrkrausestem** [community/community#138928] [GitHub Education]
### Hi everyone,

I have been using github classroom for several years now.  This year everything was going fine, I am able to create assignments using template repositories and until today the students were able to complete their assignment workflow with no issues.  Today, the students started getting the error message:

you no longer have access to your assignment repository. contact your teacher for support

I'm dead in the water and there is seemingly no way to directly contract github for support. Has anyone experienced this? Is there a fix or workaround?

**Comment by Ujoa** [comment]
same here

**Comment by Ujoa** [comment]
@github-staff

**Comment by a-tk** [comment]
I created a ticket on github support: #3003499

I will follow up if I hear anything.

**Reply by a-tk** [reply]
@mrkrausestem you can file a ticket with github at the following location: https://support.github.com/request

**Comment by chrissphinx** [comment]
Bumping this 🤷

**Reply by jeffbcit** [reply]
Having the same issue

**Comment by a-tk** [comment]
For anyone who doesn't know, there is a workaround. 

Access the students repository, and add their github username back as a collaborator with the appropriate permissions. 

This is probably scriptable with the github API for large numbers of students, but I haven't gone that far yet. Even for that to work, all of your students would need to accept the assignment, and then be locked out.

**Comment by a-tk** [comment]
My ticket was responded too indicating the issue is fixed. I can no longer reproduce the problem. 

Additionally, the team is not able to retroactively fix the repositories (that is, the ones already created that students have lost access too). You'll need to manually fix those yourselves.

**Reply by Tobeyforce** [reply]
I got the same issue now happening one year later...

**Reply by jacksonlewis97** [reply]
Yup, this just popped up for me again this week. Can't seem to find anyone who has a fix yet.

**Reply by a-tk** [reply]
Did you file a ticket with support? That's what they responded to last year. 
Fix is the same as last year, remove the student from the repository and reinvite them as a collaborator.

**Reply by jacksonlewis97** [reply]
Yeah I went ahead and put in a ticket. I was able to get the repo for one student working by reinventing them and having them accept, but the issue is happening at a scale that makes that solution unmanageable. I will update this when I get a response on my ticket.
